docs/core/policies.rst

```diff
@@ -148,7 +148,7 @@ expected outcome in the case of a tie. They look like this, where higher numbers
     | 4. ``FallbackPolicy`` and ``TwoStageFallbackPolicy``
     | 3. ``MemoizationPolicy`` and ``AugmentedMemoizationPolicy``
     | 2. ``MappingPolicy``
-    | 1. ``TEDPolicy``, ``EmbeddingPolicy``, ``KerasPolicy``, and ``SklearnPolicy``
+    | 1. ``TEDPolicy``, ``EmbeddingPolicy`` and ``KerasPolicy``
```

Comment:
In your comment you mentioned that you removed `EmbeddingPolicy`, but it is still listed. Do you want to remove it?

Reply:
I guess not. Since we still list it.